### DISCUSSION TITLE
Add Semantic Matching dataset QQP. 添加Semantic Matching数据集QQP

### DISCUSSION THREAD

**Discussion by vansin** [open-compass/opencompass#316] [Community Task]
# Add Semantic Matching dataset QQP. 
 添加Semantic Matching数据集QQP 

Please read [https://github.com/InternLM/opencompass/issues/248](https://github.com/InternLM/opencompass/issues/248) 

请认真阅读 [https://github.com/InternLM/opencompass/issues/248](https://github.com/InternLM/opencompass/issues/248) 

## Task Information

**Repository**：OpenCompass

**Difficulty**：Medium

**Points**：30

**Estimated Time Consumption**：3 days

## Task Status

**Claimant's Github ID**：None

**Claim Time**：None

**PR Link**：None

## Points to Note

在评论区回复 `Claim the mission` 并填写[问卷](https://www.wjx.top/vm/YRBjOoS.aspx#)领取任务，填写[问卷](https://www.wjx.top/vm/YRBjOoS.aspx#)后请及时添加 OpenCompass 小助手微信（微信ID：`van-sin`）获取报名结果~ 

完成任务后提交 PR 标题请以 [SIG] 开头，提交后将 PR 链接回复到本 Disucssion 评论区，我们会第一时间审核您的贡献。

**Comment by KevinNuNu** [comment]
Claim the mission

**Reply by KevinNuNu** [reply]
https://github.com/open-compass/opencompass/pull/438
